sns/parity_math.c

```diff
@@ -67,11 +78,28 @@ static void xor_recover(struct m0_parity_math *math,
                         struct m0_buf *fails,
 			enum m0_parity_linsys_algo algo);
 
+#ifndef __KERNEL__
```

Comment:
If integrated with previous patch, this should be:
#ifdef HAVE_ISAL
?

Reply:
Yes. I was thinking of doing it after closing [PR-416](https://github.com/Seagate/cortx-motr/pull/416). I would replace this with 
`#if !defined(__KERNEL__) && defined(HAVE_ISAL)`